Src/PerformanceCollector/NetCore/Perf.NetCore.csproj

```diff
@@ -28,7 +28,7 @@
 
```

Comment:
The `VersionPrefix` was missed [Line 17].
Not your fault, looks like i didn't bump it for 2.7.2.